src/python/WMCore/ReqMgr/Service/RequestAdditionalInfo.py

```diff
@@ -174,6 +191,11 @@ def _validate_args(self, param, safe):
             safe.kwargs["name"] = param.args[1]
             param.args.pop()
             param.args.pop()
+        elif args_length == 2 and param.args[0] == "update_only":
```

Comment:
Just for my education, is this validation called for any http method? or only GET?

Reply:
It call for all the call. but it will throw an error for the post call since it accepts only one argument. I will update the code to make it more clear